src/client.cpp

```diff
@@ -301,6 +301,39 @@ bool IsTimeshifting(void)
   return tvh->DemuxGetTimeshiftTime() != 0;
 }
 
+static time_t ConvertMusecsToTime(int64_t musecs)
+{
+  // tvheadend reports microseconds for timeshifting values,
+  // Kodi expects it to be an absolute UNIX timestamp
+  return time(NULL) - static_cast<time_t>(static_cast<double>(musecs / 1000000));
+}
+
+time_t GetPlayingTime()
```

Comment:
@FernetMenta you are right. Not introduced by this PR (at least not for the "old" functions), but should be fixed. Will do.